internal/x/exec/runner.go

```diff
@@ -380,7 +380,12 @@ func (r *runner) ListLintGroup(group string) error {
 }
 
 func (r *runner) ListAllLintGroups() error {
+	groups := make([]string, 0, len(lint.GroupToCheckers))
 	for group := range lint.GroupToCheckers {
+		groups = append(groups, group)
+	}
+	sort.Strings(groups)
```

Comment:
Adding this for determinism?

Reply:
Yea, cuz I'll be testing this in #74 and you want the output to be deterministic. Even without wanting this for testing, it's just nice to have the output sorted.